### PR TITLE
Add Kubernetes Page

### PR DESCRIPTION
**Pull request by mhoppal**

### CODE REVIEW

File: docs/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Monasca can be deployed into Kubernetes to monitor containers, services and Kubernetes itself.
+
+## Monitoring Kubernetes
+
+A deamonset/deployment of the [monasca-agent][4] will be brought up when deploying Monasca into Kubernetes. The
+daemonset will monitor pod/node workloads and the deployment will monitor Kubernetes status/health. They both do
+autodetection on Prometheus endpoints to scrape metrics from and send into the Monasca pipeline.
+
+<object class="img-responsive"
```
Comment (timothyb89) [inline docs/kubernetes.md]: oops, I sent you a bad example, this should be `<img class=...`

File: docs/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+These annotations being:
+
+* prometheus.io/scrape: Only scrape services that have a value of 'true'
+* prometheus.io/path: If the metrics path is not '/metrics' override this.
+* prometheus.io/port: Scrape the service on the indicated port instead of the default of '9102'.
+
+If the service contains the scrape annotation the agent will attempt to scrape metrics from it to send through Monasca.
+
+## Installing Monasca via Helm
```
Comment (timothyb89) [inline docs/kubernetes.md]: we should think about moving the content from here on to a separate page (and I should do the same with docker)

Comment (mhoppal) [reply docs/kubernetes.md]: how do you want to handle that? like kubernetes/docker deploy and kubernetes/docker monitoring?

File: docs/kubernetes.md (hunk 1 above)
Comment (mhoppal) [reply docs/kubernetes.md]: will change

File: docs/kubernetes.md (hunk 2 above)
Comment (timothyb89) [reply docs/kubernetes.md]: maybe `/docs/deploy-kubernetes.md` and `/docs/deploy-docker.md`? then `/docs/kubernetes.md` can be for the overview

Comment (timothyb89) [reply docs/kubernetes.md]: which I guess means I should write a `docker.md` :)

**Review by timothyb89 — APPROVED** [review]

**Pull request merged**